ball.py

```diff
@@ -1,10 +1,76 @@
 from random import randint, random
+from vector import Vector
 from sketchy import Sketchy
 import math
 
 class Ball(object):
+    def __init__(self, width, height):
+         self.location = Vector(width/2, height/2)
+         self.velocity = Vector(0, 0)
+        #  self.center = Vector(width/2, height/2)
```

Comment:
I don't know what you mean by setting up a specific test directory and implementing my vector tests.